### PR TITLE
added a lastName input to signUpForm

### PR DESCRIPTION
**Pull request by michaelyaacoub**
under issue: https://github.com/tdowds/rememberthee-website/issues/17

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -25,11 +25,12 @@ def get_remember_thee():
 def sign_up():
 	try:
 		data = request.get_json()
-		subject = "User Sign Up: " + data['name']
+		subject = "User Sign Up: " + data['firstName', 'lastName']
```
Comment (tdowds) [inline app.py:28]: I don't believe this is proper syntax for [accessing](https://www.w3schools.com/python/python_dictionaries_access.asp) a python dictionary. Should probably be something like:
```
subject = "User Sign Up: " + data['firstName'] + " " + data['lastName']
```

**Comment by tdowds** [comment]
Looks pretty good @michaelyaacoub ! Only comment for you above, but please be sure to test this out end-to-end (E2E) to make sure everything is working appropriately. If you need help with that, just let me know. thanks

File: app.py (hunk 1 above)
Comment (michaelyaacoub) [reply app.py]: Totally! makes sense. thanks for reviewing

**Comment by tdowds** [comment]
Looks good to me. Thanks @michaelyaacoub

**Pull request merged**